src/lib/components/forms/form/terra-form.component.ts

```diff
@@ -107,9 +106,7 @@ export class TerraFormComponent implements ControlValueAccessor, OnChanges, OnIn
      * @description Initializes the controlTypeMap with its default value if `inputControlTypeMap` is not given.
      */
     public ngOnInit(): void {
-        if (isNullOrUndefined(this.inputControlTypeMap)) {
-            this._controlTypeMap = new TerraFormTypeMap();
-        }
```

Comment:
Hierüber kann ggf. nochmal diskutiert werden. Ob wir hier garkeinen Standard setzen oder vielleicht die aktuelle FormTypeMap als Standard nehmen.

Reply:
Ich finde keinen Standard okay, denn hier wird eh sehr oft individualisiert. Das sollten wir ggf. aber dann trotzdem in den Changelog schreiben.

Reply:
um die Erwartungen an das Standardverhalten zur bewahren (ggf. ShopBuilder), könnte die `FormTypeMap` weiterhin genutzt werden. mir fehlt die Referenz zu anderen Frameworks, um das etwas ableiten zu können.
eine Meinung von @felixdausch für die Anforderungen aus dem ShopBuilder könnten hilfreich sein.

Reply:
Wie schon gesagt, hier wird oft individualisiert/eine eigene FormTypeMap genutzt. So auch im ShopBuilder.
Wenn jemand nicht individualisieren möchte, kann er unsere Standardmap (`FormTypeMap`) nutzen.
Indem wir diese aber nicht als Standard setzen werden wir flexibler. Sonst wäre nämlich jede Änderung dahingehend ein breaking change.

Reply:
Also du hast meinen Zuspruch für die Änderung. Wir sollten es mal mit dem Rest diskutieren.

Reply:
Im Kontext des ShopBuilders sehe ich da kein Problem. Wir haben letzte Woche die Terra-Form aber ohne explizite Angabe einer TypeMap verbaut:
https://github.com/plentymarkets/terra/pull/4988/files

Wir können das natürlich anpassen. Meine Erwartungshaltung wäre allerdings, dass hier ein sinnvoller Standardwert vorhanden ist. Alternativ könnte man ein entsprechendes Warning in den Log schreiben, wenn keine TypeMap angegeben wird.